examples/git-credentials/ssh-keys-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
And this file?

Reply:
That's the secret file for the ssh public key & private key. The public key is added to the git-server pod, the private key is used to push to the git server by the git-push job.